### PR TITLE
Siemens mMR NEMA IQ lowcounts

### PR DESCRIPTION
**Pull request by KrisThielemans**
Same data as the high count NEMA IQ, just unlisted with shorter duration.

This needs a different penalty though.

This PR will be rebased on `main` once #107 is merged.

### CODE REVIEW

**Comment by KrisThielemans** [comment]
@casperdcl this one is ready as well

**Comment by KrisThielemans** [comment]
data is now on https://petric.tomography.stfc.ac.uk/data/. I cannot add the BSREM plots here (until Monday)

File: petric.py [hunk 1]
```diff
@@ -248,6 +248,7 @@ def get_image(fname):
 
 DATA_SLICES = {
     'Siemens_mMR_NEMA_IQ': {'transverse_slice': 72, 'coronal_slice': 109},                 # 'sagittal_slice': 89
```
Comment (casperdcl) [inline petric.py:250]: Should we add it here too?


```suggestion
    'Siemens_mMR_NEMA_IQ': {'transverse_slice': 72, 'coronal_slice': 109, 'sagittal_slice': 89}
```

**Review by casperdcl — APPROVED** [review]

File: petric.py (hunk 1 above)
Comment (KrisThielemans) [reply petric.py:250]: Could do, but the pngs on the website didn't have it. Maybe less confusing to keep it as is.

**Comment by KrisThielemans** [comment]
# Output of `plot_BSREM_metrics`
## BSREM objective values
Siemens_mMR_NEMA_IQ_lowcounts_BSREM_objectives
Siemens_mMR_NEMA_IQ_lowcounts_BSREM_objectives_last
## metrics
Siemens_mMR_NEMA_IQ_lowcounts_metrics_BSREM_OSEM
Siemens_mMR_NEMA_IQ_lowcounts_metrics_BSREM

pass at 4160
## images
image at pass index
Siemens_mMR_NEMA_IQ_lowcounts_image_at_0 01_0 005
diff with OSEM
Siemens_mMR_NEMA_IQ_lowcounts_OSEM_diff_image_at_0 01_0 005
diff with ref
Siemens_mMR_NEMA_IQ_lowcounts_ref_diff_image_at_0 01_0 005

**Pull request merged**